src/main/java/com/spotify/github/v3/clients/RepositoryClient.java

```diff
@@ -49,9 +51,11 @@
 import java.lang.invoke.MethodHandles;
 import java.util.Iterator;
 import java.util.List;
+import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionException;
+import javax.ws.rs.core.HttpHeaders;
```

Comment:
There is an alternative class which we could use `com.google.common.net.HttpHeaders`

Reply:
I just copied this from other classes in this project, doesn't matter much I think 🤷